### ISSUE TITLE
Grant access to hypershift1 NESE pool to ESI nese-storage VLAN

### ISSUE THREAD

**Issue by larsks** [nerc-project/operations#857]
TL;DR: Can we update the network ACLs on the hypershift NESE pool so that I can access it from the ESI environment? I've sent the same request to help@nese, but my recollection is that this may be something @jtriley can do as well.

---

While running a test hosted control plane deployment yesterday, I ran into a couple of problems:

- It looks like HCP requires a minimum of three worker nodes (affinity rules on etcd pods prevent them from deploying if there are fewer than three nodes). An expedient solution would be to add a third node to the hypershift cluster, but this can't from the ESI environment because new cluster nodes require access to ports on the cluster that aren't exposed by the existing public-facing load balancer.

  If there are nodes available in the NERC environment that are not currently deployed on an existing cluster that might the quickest solution, but the nature of the NERC environment means that scaling is always going to be difficult since there is no API for managing the bare metal nodes.

- The NERC nodes have very minimal local disk space. The hosted control plane documentation strongly suggests using local (ideally SSD or NVME) storage for etcd on hosted clusters, but with less than 200GB local disk space on the existing nodes this would quickly become a limiting factor.

I think the most effective solution would be to redeploy in the ESI environment (the nodes have more local disk space, we have more direct control over networking, and it's possible to scale the cluster without involving requests to networking, etc). I've gone ahead and built out a new cluster, but it needs storage.

Rather than creating a new allocation, I would like to use the storage we've already allocated for this purpose.

**Comment by larsks** [comment]
This was resolved via the NESE help ticket.
